front_end/app/src/components/game/PongBall.css

```diff
@@ -2,12 +2,12 @@
 	--positionX: 50;
 	--positionY: 50;
 	position: absolute;
-	top: calc(var(--positionY) * 1vh);
-	left: calc(var(--positionX) * 1vw);
+	top: calc(var(--positionY) * 1%);
+	left: calc(var(--positionX) * 1%);
 	transform: translate(-50%, -50%);
 	
-	height: 2vh;
-	width: 2vh;
+	height: 2%;
+	width: 2%;
```

Comment:
If you dont want the ball to strech setting the width and height to 20px might be a solution, it works good for me